drools-workbench-models/drools-workbench-models-guided-dtable/src/test/java/org/drools/workbench/models/guided/dtable/backend/GuidedDTDRLPersistenceTest.java

```diff
@@ -1794,9 +1794,9 @@ public void testRHS() {
                      a2.getFieldValues()[0].getType());
 
         // examine the retract
-        ActionRetractFact a3 = (ActionRetractFact) rm.rhs[2];
-        assertEquals("retract",
-                     a3.getVariableName());
+//        ActionRetractFact a3 = (ActionRetractFact) rm.rhs[2];
+//        assertEquals("retract",
+//                     a3.getVariableName());
```

Comment:
This is the only part that I have questions about. Do we need to remove this or update?

Reply:
Good question, looking.

Reply:
Test is restored now.